app/src/main/java/org/wikipedia/bridge/CommunicationBridge.java

```diff
@@ -68,11 +67,11 @@ public CommunicationBridge(final WebView webView) {
     public void resetHtml(@NonNull Context context, @NonNull String assetFileName, @NonNull String wikiUrl) {
         String html = "";
         try {
-            final int rgbMask = 0xFFFFFF;
             html = FileUtil.readFile(WikipediaApp.getInstance().getAssets().open(assetFileName))
                     .replace("$wikiurl", wikiUrl)
-                    .replace("$bodybackground", String.format("#%06X",
-                            ResourceUtil.getThemedColor(context, R.attr.paper_color) & rgbMask));
+                    .replace("$pageLibThemeClass", Theme.getThemePageLibClass(WikipediaApp.getInstance().getCurrentTheme()))
+                    .replace("$pageLibDimImgClass", (WikipediaApp.getInstance().getCurrentTheme().isDark()) ? "pagelib_dim_images" : "");
```

Comment:
parentheses can be removed `(WikipediaApp.getInstance().getCurrentTheme().isDark())`

Reply:
Done